avodolazov/01_html/task3.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+  <HTML>
+  <HEAD>
+  <META charset="utf-8">
+  </HEAD>
+  <BODY>
+  <h1>HOW-TO: Программа на Python, Часть 1</h1>
+  <ul>
+<li><i>Программа на Python — часть 1</i></li>
+<li><i>Программа на Python — часть 2</i></li>
+<li><i>Программа на Python — часть 3</i></li>
+</ul>
+<h2>Содержание</h2>
+<h3>HOW-TO: Программа на Python, Часть 1</h3>
+<ul>
+<li><u>Наша первая программа</u></li>
+<li><u>Пример цикла For</u></li>
+</ul>
+<p>Из множества существующих языков программирования Python — один из самых лёгких в изучении.
+Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился. Хотя Python входит в
+состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка
+для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей —
+программирование графических интерфейсов (GUI). Давайте создадим простую программу прямо
+сейчас.</p>
+<h2>Наша первая программа</h2>
+<p>Наберём код в текстовом редакторе (например, в <strong>gedit</strong>). Потом мы разберёмся, что делает каждая строка и пойдём дальше.</p>
+<p>Введите эти 4 строки:</p>
+<pre>#!/usr/bin/env python
+print 'Hello. I am a python program.'
+name = raw_input("What is your name?")
+print "Hello there, " + name + "!"</pre>
+<p>Это всё. Сохраните файл как <b>hello.py</b> в любом месте. Я предлагаю сохранить его в папке
+<b>python_examples</b> в домашнем каталоге. Этот пример показывает, как просто написать программу на
+Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале <pre><b><u><i>
```

Comment:
Need to close p

Reply:
I close it on the next line. That's wrong?